### PR TITLE
SEC-164 - Further work is needed to clean up the pool-backed securities ontology

### PR DESCRIPTION
**Pull request by ElisaKendall**
Signed-off-by: Elisa Kendall <ekendall@thematix.com>

## Description

Revisions to the pool-backed and asset-backed securities ontologies include additional restructuring, additional instrument definitions, redefining some securities based on their CFI definitions, and so forth

Fixes: #1638 / SEC-164

## Checklist:

- [x] I'm familiar with the [FIBO developer quide](https://github.com/edmcouncil/fibo/blob/master/CONTRIBUTING.md#contributing-to-the-fibo-code). My contribution meets all the requirements described there.
- [x] My contribution follows the [principles of best practices for FIBO](https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md).
- [x] My changes have been reconciled with latest master and no merge conflicts remain.
- [x] This PR is related to exactly one issue. The issue is referenced by using a GitHub keyword such as "fixes", "closes", or "resolves".
- [x] Hygiene tests have been applied by a PR with "(WIP)" in title.
- [x] The issue has been tested locally using a reasoner (for ontology changes).

### CODE REVIEW

**Review by dsnewman — APPROVED** [review]

File: SEC/Debt/AssetBackedSecurities/CollateralizedDebtObligations.rdf [hunk 1]
```diff
@@ -466,12 +456,26 @@
...
...ese trust certificates are derived from the cash flows of the ABS bonds. This CDO instrument is part of a CDO issue, consisting of individual CDO instruments of a given seniority. Often referred to as tranches and slices (Investopedia). Investopedia: Similar in structure to a collateralized mortgage obligation (CMO) or collateralized bond obligation (CBO), CDOs are unique in that they represent different types of debt and credit risk. In the case of CDOs, these different types of debt are often referred to as &apos;tranches&apos; or &apos;slices&apos;. Each slice has a different maturity and risk associated with it. The higher the risk, the more the CDO pays. Further details: Collateralized Debt obligations are securitized interests in pools of - generally non-mortgage - assets. Assets - called collateral - usually comprise loans or debt instruments. A CDO may be called a collateralized loan obligation (CLO) or collateralized bond obligation (CBO) if it holds only loans or bonds, respectively. Investors bear the credit risk of the collateral. Multiple tranches of securities are issued by the CDO, offering investors various maturity and credit risk characteristics.</skos:definition>
```
Comment (rivettp) [inline SEC/Debt/AssetBackedSecurities/CollateralizedDebtObligations.rdf:469]: can we split between definition and note?

File: SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf [hunk 2]
```diff
@@ -163,77 +94,77 @@
...
+		<fibo-fnd-utl-av:synonym xml:lang="en">capital protected note</fibo-fnd-utl-av:synonym>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-sec-dbt-ab-pbs;SecuritizedDebtPool">
 		<rdfs:subClassOf rdf:resource="&fibo-sec-sec-pls;DebtPool"/>
 		<rdfs:label xml:lang="en">securitized debt pool</rdfs:label>
-		<skos:definition xml:lang="en">pool of individual debt instruments (such as loans or mortgages) that is securitized as part of the issue of some security</skos:definition>
+		<skos:definition xml:lang="en">pool of individual debt instruments (such as loans or mortgages) that that have been securitized through the issue of some security</skos:definition>
+		<skos:editorialNote xml:lang="en">We may not need this, but if we keep it we need to link it to the instruments that depend on it (see property &apos;backs&apos;, below).</skos:editorialNote>
```
Comment (rivettp) [inline SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf:128]: address this note

File: SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf [hunk 3]
```diff
@@ -163,77 +94,77 @@
...
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-sec-dbt-ab-pbs;StructuredFinanceInstrument">
-		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-dbti;TradableDebtInstrument"/>
+		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ab-pbs;PoolBackedSecurity"/>
 		<rdfs:label xml:lang="en">structured finance instrument</rdfs:label>
-		<skos:definition xml:lang="en">debt instrument that has periodic principal payments with the amount of principal that is paid being dictated by a published factor</skos:definition>
+		<rdfs:seeAlso rdf:resource="https://www.bis.org/publ/cgfs23mitchell.pdf"/>
+		<skos:definition xml:lang="en">debt instrument presented to large financial institutions or companies with complicated financing needs that are unsatisfied with conventional financial products</skos:definition>
```
Comment (rivettp) [inline SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf:135]: yuk. should not base the definition  on how it's marketed and even worse their "satisfaction"

File: SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf [hunk 4]
```diff
@@ -163,77 +94,77 @@
...
 		<skos:example xml:lang="en">Collateralized debt obligations (CDOs), synthetic financial instruments, collateralized bond obligations (CBOs), and syndicated loans are examples of structured finance instruments.</skos:example>
-		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">This may be either an invented product (synthetic) or based on a real underlying asset pool. Either way, the underlying is defined by the issuing bank or investment company. These are traded. This is based on tranches of underlying asset pools. These are defined by the issuing bank or investment company.</fibo-fnd-utl-av:explanatoryNote>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Structured finance instruments represent a form of securitization technology which can be defined by three key characteristics: (1) pooling of financial assets, such as loans, bonds, or credit-default swaps; (2) de-linking of the credit risk of the asset pool from the credit risk of the originating firm, usually through use of a finite-lived, stand-alone special purpose vehicle (SPV); and (3) issuance by the SPV of &quot;tranched&quot; liabilities backed by the asset pool.</fibo-fnd-utl-av:explanatoryNote>
```
Comment (rivettp) [inline SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf:137]: "technology" does not seem right here

**Review by rivettp — DISMISSED** [review]
OK, with edits made on the call applied

File: SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf (hunk 2 above)
Comment (ElisaKendall) [reply SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf:122]: This class will need further review, and may be eliminated but that's not clear yet.

File: SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf (hunk 3 above)
Comment (ElisaKendall) [reply SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf]: Fixed on our DER telecon today.

File: SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf (hunk 4 above)
Comment (ElisaKendall) [reply SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf]: Fixed on our DER telecon today.

**Review by rivettp — APPROVED** [review]

**Review by jfgemski — APPROVED** [review]

**Review by tahoeblue — APPROVED** [review]

**Pull request merged**